actionview/lib/action_view/helpers/tags/file_field.rb

```diff
@@ -6,6 +6,9 @@ module Tags # :nodoc:
       class FileField < TextField # :nodoc:
         def render
           include_hidden = @options.delete(:include_hidden)
+          if @options[:accept].is_a?(Array)
+            @options[:accept] = @options[:accept].join(",")
+          end
```

Comment:
Is this a [breaking change](https://guides.rubyonrails.org/contributing_to_ruby_on_rails.html#breaking-changes)? It might require introducing a configuration to control the [change in behavior](https://guides.rubyonrails.org/contributing_to_ruby_on_rails.html#changing-behavior).